features/data_gov_uk.feature

```diff
@@ -28,6 +28,7 @@ Feature: Data.gov.uk
   @notintegration @notstaging
   Scenario: Check CKAN action api's search works
     Given I am testing "ckan"
+    And I do not bypass caching
```

Comment:
```suggestion
    And I do not try to bypass caching
```

Optional: I think this is a slightly clearer as it hints at the mechanism in use; we're not sending some magical "do / don't cache signal", it's just a query param tacked onto the URL.

Reply:
Nice suggestion!